pkg/restapi/edv/operation/operations.go

```diff
@@ -429,6 +438,41 @@ func (c *Operation) logSpecHandler(rw http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (c *Operation) logSpecGetHandler(rw http.ResponseWriter, _ *http.Request) {
+	logLevels := log.GetAllLevels()
+
+	var defaultDebugLevel string
+
+	c.getLogSpecResponse.Reset()
```

Comment:
this looks strange. Would look better if `Operation` had a stringBuilder *provider* (eg. `func() stringBuilder`)

Reply:
Will incorporate in my next PR :)